BP/history/characters/romano_frank.txt

```diff
@@ -31,7 +31,9 @@
 	culture="romano_frank"
 	father=1310003
 	410.1.1={
+		birth=yes
 	}
 	460.1.1={
+		death=yes
```

Comment:
I don't know how that got added without the birth and date effects, going to fix for CK2 as well